### PR TITLE
Rate Stars from Search Modal

### PR DESCRIPTION
**Pull request by sk1122**
## Description
I have changed the way stars were displayed in Modal, Now each star has its own id and is a button, they are wrapped in form tag, whenever submits the star button, an AJAX Request is initiated and bookrating is changed in db, default rating is from csv file. 

## Affected Dependencies
No Dependencies is affected

## How has this been tested?
Django Tests have run and unit tests of Github Workflow also passes

## GIF Before
- If any UI Changes, Use [this](https://www.screentogif.com/) to record or else specify NA
- 

## GIF After

https://user-images.githubusercontent.com/40713709/110913799-95889f00-833b-11eb-8238-26d58f267ab7.mp4

## Checklist
- [ *] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [ *] My changes do not edit/add any unrequired files.
- [ *] My changes are covered by tests.

### CODE REVIEW

File: BookRecSystem/settings.py [hunk 1]
```diff
@@ -22,12 +22,12 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('KITABE_SECRET_KEY')
```
Comment (Praful932) [inline BookRecSystem/settings.py:25]: Please unstage this file from PR

File: mainapp/views_ajax.py [hunk 2]
```diff
@@ -96,6 +96,7 @@ def user_rate_book(request):
         else:
             # Update Rating
             rating_object = query[0]
+            print(rating_object)
```
Comment (Praful932) [inline mainapp/views_ajax.py:99]: Please remove debugging statements :) from PR

File: static/mainapp/css/layout.css [hunk 3]
```diff
@@ -120,6 +120,23 @@ body {
 .modal-footer {
   color: white;
 }
+<<<<<<< HEAD
```
Comment (Praful932) [inline static/mainapp/css/layout.css:123]: There's a conflict here?

File: templates/mainapp/layout.html [hunk 4]
```diff
@@ -119,8 +119,18 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
                             <strong>Authors:</strong>
                             <p id="bookauthors">
                             </p>
+                            <!-- Rating Block -->
 
-                            <div id="bookrating"></div>
+                            <div class="ratings" id="rate">
+                                <form class="rate-form" action="" method="POST">
+                                    {% csrf_token %}
+                                    <button type="submit" class="fa fa-star star-btn" id="first"></button>
```
Comment (Praful932) [inline templates/mainapp/layout.html:127]: Hi, this is a nice implementation of yours for rating, though I would recommend to reuse the implementation in [layout.html](https://github.com/Praful932/Kitabe/blob/master/templates/mainapp/layout.html#L271) :)

**Comment by Praful932** [comment]
@sk1122 Btw the commit history seems weird to me, are you using codespaces? I would suggest to name the commits according to the changes that you're making :)

**Comment by sk1122** [comment]
No, it was a issue which I was facing with git, so it is showing some weird git commit history

**Review by Praful932 — COMMENTED** [review]
> No, it was a issue which I was facing with git, so it is showing some weird git commit history

What IDE are you using?

**Comment by sk1122** [comment]
I have changed all files but they keep appearing when committing. 
I unstaged settings.py using
`git reset BookRecSystem/settings.py
`

But the problem persists when I committed again.

**Comment by sk1122** [comment]
I think it is fine now

**Comment by Praful932** [comment]
@sk1122 Could you [unstage ](https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request)all not required files from PR, Then I can review

**Review by Praful932 — CHANGES_REQUESTED** [review]
@sk1122 Could you [unstage ](https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request)all not required files from PR, Then I can review

**Comment by sk1122** [comment]
I will do it

**Comment by Praful932** [comment]
@sk1122 Any updates?

**Comment by sk1122** [comment]
I had committed yesterday, but it was creating conflict in layout.html,

I had unstaged all nit required files but conflict happened. @Praful932

**Comment by Praful932** [comment]
@sk1122 The issue seems to persist, could you open a new PR branching off master, if this is persisting

**Comment by sk1122** [comment]
Ok

**Pull request closed without merging**